### ISSUE TITLE
Share usage statistics checkbox does not appear in tab order

### ISSUE THREAD

**Issue by elharo** [GoogleCloudPlatform/google-cloud-eclipse#1129]
![image](https://cloud.githubusercontent.com/assets/1005544/21185128/651014f8-c1dd-11e6-891c-9135f99da9d7.png)

You can't tab to the first checkbox. 

Also the text here is not selectable and copyable and it should be.

**Comment by chanseokoh** [comment]
Not reproducible on Linux. We saw that this doesn't work on OSX, so maybe something should be done for OSX?

**Comment by diegomarquezp** [comment]
Closing as not planned. cc: @blakeli0
